onnx/backend/test/case/node/lstm.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class LSTM_Helper():
+    def __init__(self, **params):
+        #LSTM Input Names
+        X = 'X'
+        W = 'W'
+        R = 'R'
+        B = 'B'
+        H_0 = 'initial_h'
+        C_0 = 'initial_c'
+        P = 'P'
+
+        required_inputs = [X, W, R]
+        for i in required_inputs:
+            assert i in params, "Missing Required Input: {0}".format(i)
+
+        num_directions = params[W].shape[0]
+
+        if(num_directions == 1):
+            for k in params.keys():
+                params[k] = np.squeeze(params[k], axis=0)
+
+            hidden_size = params[R].shape[-1]
+            batch_size = params[X].shape[0]
+
+            b = params[B] if B in params else np.zeros(8 * hidden_size)
+            p = params[P] if P in params else np.zeros(3 * hidden_size)
+            h_0 = params[H_0] if H_0 in params else np.zeros((batch_size, hidden_size))
+            c_0 = params[C_0] if C_0 in params else np.zeros((batch_size, hidden_size))
+
+            self.X = params[X]
+            self.W = params[W]
+            self.R = params[R]
+            self.B = b
+            self.P = p
+            self.H_0 = h_0
+            self.C_0 = c_0
+        else:
+            raise NotImplementedError()
```

Comment:
Bidirectional LSTM?

Reply:
I agree that we need tests for bidirectional LSTM, as well as different variations of activations and etc. However, The scope for this PR is to add basic test coverage for forward case and also provide a base for numpy implantations that shall be extended later. Additional tests, which would also required extension of base numpy implementation, shall be added later.

Reply:
OK.